### ISSUE TITLE
Building buildbot-www fails with Node.js 11 and 10.14.2

### ISSUE THREAD

**Issue by yan12125** [buildbot/buildbot#4382]
With Node.js 10, running `python setup.py build` in www/base worked fine. However, with Node.js 11,
```
running build
running build_py
running build_js
yarn: 1.10.1 npm:  6.4.1
Running command: yarn install --pure-lockfile
yarn install v1.10.1
warning package.json: No license field
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
warning buildbot-www: No license field
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@1.2.4: The platform "linux" is incompatible with this module.
info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
[5/5] Building fresh packages...
Done in 238.96s.
Running command: /build/buildbot-www-git/src/buildbot/www/base/node_modules/.bin/gulp prod --notests
fs.js:25
'use strict';
^

ReferenceError: internalBinding is not defined
    at fs.js:25:1
    at req_ (/build/buildbot-www-git/src/buildbot/www/base/node_modules/guanlecoja/vendors.js:223323:5)
    at Object.req [as require] (/build/buildbot-www-git/src/buildbot/www/base/node_modules/guanlecoja/vendors.js:223240:10)
    at Object.859.natives (/build/buildbot-www-git/src/buildbot/www/base/node_modules/guanlecoja/vendors.js:114601:37)
    at s (/build/buildbot-www-git/src/buildbot/www/base/node_modules/guanlecoja/vendors.js:1:282)
    at /build/buildbot-www-git/src/buildbot/www/base/node_modules/guanlecoja/vendors.js:1:333
    at Object.860../fs.js (/build/buildbot-www-git/src/buildbot/www/base/node_modules/guanlecoja/vendors.js:114606:27)
    at s (/build/buildbot-www-git/src/buildbot/www/base/node_modules/guanlecoja/vendors.js:1:282)
    at /build/buildbot-www-git/src/buildbot/www/base/node_modules/guanlecoja/vendors.js:1:333
    at Object.856.graceful-fs (/build/buildbot-www-git/src/buildbot/www/base/node_modules/guanlecoja/vendors.js:112480:10)
/usr/lib/python3.7/site-packages/setuptools/dist.py:398: UserWarning: Normalizing '1.5.1-dev39' to '1.5.1.dev39'
  normalized_version,
error: cannot copy tree 'buildbot_www/static': not a directory
```

**Comment by tardyp** [comment]
Issue is reproduced. sounds cryptic. not sure I want to install node11 to figure it out. maybe we should at least wait node 11.1 or 11.2?

**Comment by yan12125** [comment]
Update: I got the same `internalBinding is not defined` error with Node 10.14.2, too. Maybe the breaking change is backported to Node 10.

Looking forward to https://github.com/buildbot/guanlecoja/pull/13 :)

**Comment by yan12125** [comment]
The issue has been fixed in buildbot 2.3.0. Thanks for the work!

Ref: https://github.com/buildbot/guanlecoja/commit/63250c713cef7775bdc7cda1db6ad6d9fa741a48#diff-8ee2343978836a779dc9f8d6b794c3b2L4429
